README/Album.md

```diff
@@ -7,3 +7,6 @@ This is spring full stack web application and we can run using ./gradlew bootrun
 * Created album test
 # Run
 Running using ./gradlew bootrun
+A user should be able to update information about songs on the site.

```

Comment:
Your README file isn't in a place where it's visible at the top level of your repo. It needs to be named `README.md`, and it needs to be in the root of your repository.